doc/sf/jacobi_theta.qbk

```diff
@@ -0,0 +1,349 @@
+[/
+Copyright (c) 2020 Evan Miller
+Use, modification and distribution are subject to the
+Boost Software License, Version 1.0. (See accompanying file
+LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+]
+
+[section:jacobi_theta Jacobi Theta Functions]
+
+[section:jacobi_theta_overview Overview of the Jacobi Theta Functions]
+
+The Jacobi Theta functions are a set of four inter-related periodic functions of /x/ which are expressed in terms of a parameter /q/ (also called the nome), or a closely related value, [tau]
+[footnote [@https://en.wikipedia.org/wiki/Theta_function Wikipedia: Theta function]]
+[footnote [@https://mathworld.wolfram.com/JacobiThetaFunctions.html Weisstein, Eric W. "Jacobi Theta Functions." From MathWorld - A Wolfram Web Resource.]]
+[footnote [@https://dlmf.nist.gov/20 Digital Library of Mathematical Functions: Theta Functions, Reinhardt, W. P.,  Walker, P. L.]].
+
+Mathematically, the functions are described by:
```

Comment:
"Mathematically, the functions are described by:" -> "They are" (no semicolon)

Reply:
OK